src/__stories__/config.js

```diff
@@ -0,0 +1,8 @@
+const { configure } = require("@kadira/storybook")
+
+function loadStories() {
+  require("../components/__stories__/artwork")
+  require("../components/__stories__/artwork_grid")
+}
+
+configure(loadStories, module);
```

Comment:
Yeah, so, that works, but it’s now tied to FuseBox, which is not ideal for when we deploy the static version.